Source/AppAuth/iOS/OIDExternalUserAgentIOS.m

```diff
@@ -76,7 +76,7 @@ - (nullable instancetype)initWithPresentingViewController:
 }
 
 - (BOOL)presentExternalUserAgentRequest:(id<OIDExternalUserAgentRequest>)request
-                                session:(id<OIDExternalUserAgentSession>)session {
+                                session:(id<OIDExternalUserAgentSession>)session NS_EXTENSION_UNAVAILABLE("Uses APIs (i.e UIApplication.sharedApplication) not available for use in App Extensions.") {
```

Comment:
I think we might want to move this to the [method declaration in the OIDExternalUserAgent protocol](https://github.com/openid/AppAuth-iOS/blob/e191b1beadf3041259652e6e464eaceff16e5fbb/Source/AppAuthCore/OIDExternalUserAgent.h#L41).